src/adapters/MeetingsJSONAdapter.js

```diff
@@ -384,6 +394,28 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
     return captureStream;
   }
 
+  /**
+   * Returns available media devices.
+   *
+   * @param {string} type String specifying the device type.
```

Comment:
double space. And the type may be `{'videoinput'|'audioinput'|'audiooutput'}` and you can add a note like:

> See https://developer.mozilla.org/en-US/docs/Web/API/MediaDeviceInfo/kind